projects/canopy/src/lib/button/button.component.scss

```diff
@@ -24,7 +24,10 @@
 
   &:focus,
   &:hover:focus {
-    @include lg-outer-focus-outline(var(--default-focus-color));
+    @include lg-new-outer-focus-outline(
```

Comment:
Maybe we could call this `lg-btn-outer-focus-outline` so that it's a bit more specific.

Reply:
Or actually it could be just `lg-focus-outline` as this will become the main focus for everything I believe.